packages/gatsby/src/bootstrap/load-plugins/load.js

```diff
@@ -159,5 +158,29 @@ module.exports = async (config = {}) => {
     },
   })
 
+  // Add the auto page creator plugin
+  const pluginPageCreator = {
+    version: require(
+      path.join(
+        path.dirname(
+          require.resolve(`gatsby-plugin-page-creator`)), `package.json`
+        )
+      ).version,
+
+    path: slash(
+      path.dirname(require.resolve(`gatsby-plugin-page-creator`))
+    ),
+  }
+  plugins.push({
+    resolve: pluginPageCreator.path,
+    id: `Plugin gatsby-plugin-page-creator`,
+    name: `gatsby-plugin-page-creator`,
+    version: pluginPageCreator.version,
+    pluginOptions: {
+      plugins: [],
+      path: `${process.cwd()}/src/pages`,
```

Comment:
Instead of `process.cwd` use `program.directory`. Also use path.join to ensure cross-os compatability.

Reply:
How do I get `program.directory`?

Reply:
Check here - https://github.com/gatsbyjs/gatsby/blob/a38f49efcfdb79b86e0b646e3bd512a61d46e5f4/packages/gatsby/src/utils/js-chunk-names.js#L10-L15